### DISCUSSION TITLE
A Little Help?

### DISCUSSION THREAD

**Discussion by PsychoxCupcake** [AveYo/MediaCreationTool.bat#27] [Q&A]
![Screenshot 2021-10-29 224532](https://user-images.githubusercontent.com/93416684/139519328-5a7f23dd-8e94-4c28-8de1-b3f291d774f3.png)
Hi all, I'm new so if this is a stupid question then I'm sorry. I was just wondering how I get around this? Did I do something wrong? Is there something I need to do differently or is there a way to fix it? Also, will it still bypass the TPM requirement?

**Comment by AveYo** [comment]
should be fixed as of today, since I've added download methods fallback 
_(script used to have them a few years ago, but then it was not needed anymore... until 11 came)_
